changelog/4280.bugfix.rst

```diff
@@ -0,0 +1 @@
+Handle quitting from pdb with ``--trace`` by exiting pytest.
```

Comment:
Sorry if this is obvious, but I don't quite get what this means. 😁

Reply:
Using `q` in pdb should exit tests.
Currently it will run all remaining tests afterwards.
This is similar to previous changes from me in this regard.
It needs more work though (adjusting/verifying tests).

Reply:
Let me know if this is still unclear.
I agree that it could be a bit more verbose.
This improves on 86c7dcff.

Reply:
Now I understand, thanks!

Perhaps mentioning the old behavior will help to make it clear:

> Pressing `q` when in pdb will now exit pytest, instead of continue to run the remaining tests.

Reply:
Added, thanks.

Please review the tests carefully (we have not coverage for those).

Reply:
I have played around with the tests and it seems to me they are behaving well. 👍